### PR TITLE
Monero full node on ubuntu

### PR DESCRIPTION
**Pull request by luigi1111**
This commit includes a full build script for deploying a Monero full node on Ubuntu 14.04 (and possibly higher), as well as all deployment metadata.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@luigi1111__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

File: monero-on-ubuntu/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+          "description": "Hostname assigned to the Public IP, also used as the VM Name. Must be lowercase. It should match with the following regular expression: ^[a-z][a-z0-9-]{1,61}[a-z0-9]$ or it will raise an error."
+       }
+    },
+    "diskName": {
+       "type": "string",
+       "metadata": {
+          "description": "Enter a name for the OS disk; can be anything, but must be unique."
+       }
+    },
+    "installMethod": {
```
Comment (singhkay) [inline monero-on-ubuntu/azuredeploy.json]: This should become a variable. There's only one allowed value for this. No reason to ask the user for this until there's another allowed value.

File: monero-on-ubuntu/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+        "Download_Binaries"
+      ],
+      "metadata": {
+        "description": "Method to install Monero. Download_Binaries: download official binaries"
+      }
+    },
+    "vmSize": {
+      "type": "string",
+      "defaultValue": "Standard_A1",
+      "allowedValues": [
```
Comment (singhkay) [inline monero-on-ubuntu/azuredeploy.json]: Use D*_v2 sizes here. For e.g. Standard_D1_v2

**Comment by luigi1111** [comment]
Thanks. I guess I made "V2" capitalized. Should I fix that?

**Comment by jedigras** [comment]
I tested with V2 capitalized just now.  Works perfect.

**Comment by singhkay** [comment]
@jedigras Thanks for testing! lgtm, merged

**Pull request merged**